doc/how-to/preseed.yaml

```diff
@@ -29,6 +29,7 @@ ovn:
   ipv4_gateway: 10.0.0.1/24
   ipv4_range: 10.0.0.100-10.0.0.254
   ipv6_gateway: cafe::1/64
+  dns_servers: 10.0.0.1,cafe::1
```

Comment:
Just out of interested, does the order matter here? I saw it in https://github.com/canonical/microcloud/pull/269 too.

Reply:
In the LXD OVN driver, [we split the list by ipv4 and ipv6](https://github.com/canonical/lxd/blob/01f0097066abd69f9b7d53f2b2278f724998a1ed/lxd/network/driver_ovn.go#L1125-L1143) so I I don't believe the order matters.